packages/lib/src/types/index.ts

```diff
@@ -97,6 +97,26 @@ export interface PaymentMethod {
      * Configuration props as set by the merchant in the CA and received in the PM object in the /paymentMethods response
      */
     configuration?: object;
+
+    /**
+     * Brand for the selected gift card. For example: plastix, hmclub.
+     */
+    brand?: string;
+
+    /**
+     * List of possible brands. For example: visa, mc.
+     */
+    brands?: string[];
+
+    /**
+     * The funding source of the payment method.
+     */
+    fundingSource?: string;
+
+    /**
+     * The group where this payment method belongs to.
+     */
+    group?: object;
```

Comment:
As we know the possible shape of the `group` object according to the API documentation, can you define it here instead of using the type `object`?

`object` is more generic and implies that it can have different properties from time to time. That isn't the case for the `group` property.